components/PageCard.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { Flex, Text } from '@100mslive/react-ui';
+
+import { DynamicIcon } from './Callout';
+import { ArrowRightIcon } from '@100mslive/react-icons';
+
+const PageCard = ({
+    showCta = true,
+    showTag = false,
```

Comment:
This can be renamed to tag and it can be used in place of "Tag Text". If this is not present, we don't need to render the tag